flink-filesystems/flink-hadoop-fs/src/main/java/org/apache/flink/runtime/util/HadoopUtils.java

```diff
@@ -44,7 +44,9 @@
 
 	public static Configuration getHadoopConfiguration(org.apache.flink.configuration.Configuration flinkConfiguration) {
 
-		Configuration result = new Configuration();
+		// the HdfsConfiguration adds per default hdfs-site.xml and hdfs-default.xml
+		// as a default resource which is read from the class path
+		Configuration result = new HdfsConfiguration();
```

Comment:
It's not mandatory to load in HDFSConfiguration, as trying to connect to HDFS via FileSystem.get(hdfs://) kicks all this off.

Reply:
Not sure whether this may actually be called early (before access to "hdfs://") or if there is caching involved. If it does not hurt, I'd say let's keep instantiating the `HdfsConfiguration`.